examples/todo-app-tutorial/src/main.ts

```diff
@@ -1,23 +1,28 @@
 import {
-    FluentDesignSystemProvider,
-    FluentButton,
-    FluentCheckbox,
-    FluentTextField,
-    FluentCard,
-    FluentSlider,
-    FluentSliderLabel,
+    fluentButton,
+    fluentCard,
+    fluentCheckbox,
+    fluentDesignSystemProvider,
+    fluentSlider,
+    fluentSliderLabel,
+    fluentTextField,
 } from "@fluentui/web-components";
+import { DesignSystem } from "@microsoft/fast-foundation";
 import { TodoApp } from "./todo-app";
 import { TodoForm } from "./todo-form";
 import { DesignPropertyPanel } from "./design-property-panel";
 
-FluentDesignSystemProvider;
-FluentButton;
-FluentCheckbox;
-FluentTextField;
-FluentCard;
-FluentSlider;
-FluentSliderLabel;
+DesignSystem.getOrCreate()
```

Comment:
Let's update to use the new API `provideFluentDesignSystem()` and remove the `withPrefix(...)` calle.